scripts/build-all-examples/src/main.rs

```diff
@@ -8,11 +8,18 @@ use std::{
     path::{Path, PathBuf},
 };
 
+fn get_sway_path() -> PathBuf {
+    let mut curr_path = std::env::current_dir().unwrap();
+    loop {
+        if curr_path.ends_with("sway") {
+            return curr_path;
+        }
+        curr_path = curr_path.parent().unwrap().to_path_buf()
+    }
+}
```

Comment:
Instead, can the program accept a list of paths? That way it can even be used outside the example directory for things like building and testing the stdlib!